WordPress/src/main/java/org/wordpress/android/ui/mysite/cards/quicklinksitem/QuickLinkRibbonViewHolder.kt

```diff
@@ -34,8 +34,5 @@ class QuickLinkRibbonViewHolder(
 
     fun bind(quickLinksItem: QuickLinksItem) = with(binding) {
         (quickLinksItemList.adapter as QuickLinksItemAdapter).update(quickLinksItem.quickLinkItems)
-        if (quickLinksItem.showMoreFocusPoint) {
```

Comment:
❓ @AjeshRPai - Isn't this needed to scroll to More for quick Start?

Reply:
Hey @zwarm 
I tested the quick start focus point scroll in the same way you described above by making the quick link off screen and then starting the stats,pages or sharing quick start task, I was scrolled to the correct position. I removed the logic of scrolling inside the quick links because i felt that it was added when the view was quick link ribbon(horizontal) and we just didn't remove that logic when we converted the view to a vertical view.  


> I fear I know the answer, but wanted to check with you first.
I am not sure what you mean here to be honest

Reply:
> I am not sure what you mean here to be honest

That it not scrolling to "more" was a compromise. On my pixel 5x, it doesn't scroll to More if more is off screen.